core-bundle/src/Resources/contao/library/Contao/System.php

```diff
@@ -460,7 +460,7 @@ public static function loadLanguageFile($strName, $strLanguage=null, $blnNoCache
 		}
 
 		// Return if the language file has been loaded already
-		if (!$blnNoCache && isset(static::$arrLanguageFiles[$strName][$strLanguage]))
+		if (!$blnNoCache && array_key_last(static::$arrLanguageFiles[$strName] ?? array()) === $strLanguage)
```

Comment:
`array_key_last` is only available in PHP 7.3, but Contao 4.9 supports lower versions

Reply:
Fixed in a2171c6ff1afe09db87c81fd8bf141f08e7b60ad